### PR TITLE
[feature](http) refactor version info and add new http api for get version info

### PR DESCRIPTION
**Pull request by nextdreamblue**
# Proposed changes

Issue Number: close #xxx

## Problem summary

当前的doris的version信息就是一个可以自定义的字符串，对于有进行版本信息比较的场景不太友好。目前根据之前的命名规则，以及可能得其他信息，对version信息进行了拆分，分成了prefix、major、minor、patch、rc信息等几个单独的信息。
并且在fe和be上各增加了两个http接口，方便用户获取与版本相关的所有信息，用户可以根据这些信息去进行版本判断、分析、比较等行为。

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] No
2. Has unit tests been added:
    - [ ] No
3. Has document been added or modified:
    - [ ] Yes
4. Does it need to update dependencies:
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hf200012** [comment]
It is recommended that the current version can be viewed in the start script of fe and be
For example.
sh bin/start_fe.sh --version
sh bin/start_be.sh --version

**Comment by nextdreamblue** [comment]
> It is recommended that the current version can be viewed in the start script of fe and be For example. sh bin/start_fe.sh --version sh bin/start_be.sh --version

if client want get version info, the script is not simple to use.

File: docs/en/docs/admin-manual/http-actions/fe/fe-version-info-action.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+---
+{
+    "title": "Fe Version Info Action",
+    "language": "zh-CN"
```
Comment (stalary) [inline docs/en/docs/admin-manual/http-actions/fe/fe-version-info-action.md:4]:
```suggestion
    "language": "en"
```

File: docs/en/docs/admin-manual/http-actions/be-version-info-action.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
+---
+{
+    "title": "Be Version Info Action",
+    "language": "zh-CN"
```
Comment (stalary) [inline docs/en/docs/admin-manual/http-actions/be-version-info-action.md:4]:
```suggestion
    "language": "en"
```

File: gensrc/script/gen_build_version.sh [hunk 3]
```diff
@@ -27,7 +27,13 @@
 
 set -eo pipefail
 
-build_version="trunk"
+build_version_prefix="doris"
```
Comment (stalary) [inline gensrc/script/gen_build_version.sh:30]: Does it need to be manually modified when we publish the version?

**Review by stalary — CHANGES_REQUESTED** [review]

File: gensrc/script/gen_build_version.sh (hunk 3 above)
Comment (nextdreamblue) [reply gensrc/script/gen_build_version.sh:30]: yes, it needs modify version info manually in this commit

Comment (stalary) [reply gensrc/script/gen_build_version.sh:30]: Can it be specified at compile?

Comment (nextdreamblue) [reply gensrc/script/gen_build_version.sh:30]: you can specify any version info you want in this script when you complie

**Review by lide-reed — APPROVED** [review]
LGTM

**Review by stalary — APPROVED** [review]
LGTM

**Pull request merged**